src/main/java/fi/metatavu/soteapi/wordpress/events/EventUpdateTaskModel.java

```diff
@@ -0,0 +1,72 @@
+package fi.metatavu.soteapi.wordpress.events;
+
+import java.io.Serializable;
+
+public class EventUpdateTaskModel implements Serializable {
+
+  private static final long serialVersionUID = -8595848061651103341L;
+  
+  private String title;
+  
+  private String description;
+
+  private String originId;
+
+  private String slug;
+
+  private String category;
+
+  private String startTime;
+
+  private String endTime;
+
+  private Boolean allDay;
+  
+  public EventUpdateTaskModel() {
+  }
+
+  public EventUpdateTaskModel(String originId, String title, String description, String slug, String category, String startTime, String endTime, Boolean allDay) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)